support/@trustfractal/data-host/src/lib/fractalUI.ts

```diff
@@ -0,0 +1,11 @@
+// import App from '@components/App';
+import ReactDOM from 'react-dom';
+
+import App from '../popup/components/App';
+
+export class FractalUI {
+  // substrateAddress is going to be used to send funds
```

Comment:
There's no substrate address mentioned here. If this should be a TODO, please prefix with `TODO(melatron): ...`